cardano-testnet/test/cardano-testnet-test/Cardano/Testnet/Test/Cli/Query.hs

```diff
@@ -439,6 +439,10 @@ hprop_cli_queries = integrationWorkspace "cli-queries" $ \tempAbsBasePath' -> H.
       -- treasury
       H.noteM_ $ execCli' execConfig [ eraName, "query", "treasury" ]
 
+    TestQueryProposalsCmd -> do
```

Comment:
@palas can you take the lead on this.

Reply:
Related PR: https://github.com/IntersectMBO/cardano-node/pull/6045 - idk if @smelc is working on it atm.

Reply:
Addressed on: https://github.com/IntersectMBO/cardano-node/pull/6068/commits/1e0ab9ac92c83145581ad37bce2d43937f0f28b0